Automattic/jetpack
# Instant Search: Improve build process

Fixes #14405 
#### Changes proposed in this Pull Request:
* Removes using gulp for building Jetpack Search stylesheets.
* Generate the necessary stylesheet using Webpack. 

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
None.

#### Testing instructions:
* Run `yarn build-search`; ensure that it builds as expected without any issues.
* Apply this patch to your Jetpack site with Instant Search enabled.
* Try searching the site through the Customizer interface, toggling between Minimal and Expanded search result types. Ensure that everything works as expected.

#### Proposed changelog entry for your changes:
* None.
